is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
+      if (elementToCompare === currentElement) {
+        isAllUnique = false;
+      }
+    });
+  });
+
+  return isAllUnique;
```

Comment:
Good job writing this algorithm. This would run but there might be way to find whether all elements in an array is unique a little more efficient by switching the isAllUnique to false when you find a duplicated value and end the iteration there instead of iterated through all elements of the array